### ISSUE TITLE
Add login

### ISSUE THREAD

**Issue by adantoscano** [axa-group/nlp.js-app#56]
**Is your feature request related to a problem? Please describe.**
When deploy app to public, this can be access by everyone

**Describe the solution you'd like**
Add login page to client and auth middleware to Hapi server

**Describe alternatives you've considered**
Set an auth redirect in your webserver

**Comment by ericzon** [comment]
Yes, this and the import / export are one of the most interesting next steps. I'll try to gather some time for this.
